src/parser/jobs/brd/modules/Snapshots.js

```diff
@@ -47,13 +48,46 @@ export default class Snapshots extends Module {
 				// So we can show they did cast it, I'm going to generate an empty snapshot
 				snapshot = {statuses: []}
 			}
-			const snapshotCell = <Table.Cell>
+
+			const dotStatuses = [
+				STATUSES.CAUSTIC_BITE.id,
+				STATUSES.STORMBITE.id,
+				STATUSES.VENOMOUS_BITE.id,
+				STATUSES.WINDBITE.id,
+			]
+
+			const personalStatuses = [
+				STATUSES.MEDICATED.id,
+				STATUSES.RAGING_STRIKES.id,
+			]
+
+			// Sort personal buffs to the front of the status list (why do I have to cast this to Array???? JS explain??)
+			Array(snapshot.statuses).some((status, index) => {
```

Comment:
`event.snapshot` comes from `brd/AdditionalStats`, it's not a standard CastEvent prop.
From the looks of it these statuses are actually not arrays, they are key/value pairs (objects) where the key is the ID of a status.
For this you could try [ Object.entries() ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) and it might be a little safer than casting it.